pkg/ocm/versions_test.go

```diff
@@ -57,6 +58,39 @@ var _ = Describe("Versions", Ordered, func() {
 	})
 })
 
+var _ = Describe("Minimal http tokens required version", Ordered, func() {
+
+	Context("validate http tokens reuqired version", func() {
```

Comment:
This tests are similar to the version ones you added in helpers_test.go, why not group them together here?

Reply:
You are right, i remove this part of helpers. I believe each file should have tests that are relevant only to it.